knex/index.d.ts

```diff
@@ -23,7 +22,8 @@ interface Knex extends Knex.QueryInterface {
 
     raw: Knex.RawBuilder;
     transaction: <R>(transactionScope: ((trx: Knex.Transaction) => void)) => Promise<any>;
```

Comment:
This isn't your change, but it caught my attention -- this ```<R>``` is unnecessary and can be removed.